### PR TITLE
docs: add sample for docker credential store

### PR DESCRIPTION
**Pull request by sajayantony**

### CODE REVIEW

**Review by shizhMSFT — APPROVED** [review]
LGTM

File: docs/Client_Libraries/go.mdx [hunk 1]
```diff
@@ -241,4 +241,79 @@ func main() {
 	}
 }
 
+```
+
+## Pull an Image using the Docker credential store
```
Comment (shizhMSFT) [inline docs/Client_Libraries/go.mdx:246]:
```suggestion
### Pull an Image using the Docker credential store
```

**Review by shizhMSFT — DISMISSED** [review]
LGTM

File: docs/Client_Libraries/go.mdx [hunk 2]
```diff
@@ -241,4 +241,79 @@ func main() {
 	}
 }
 
+```
+
+### Pull an Image using the Docker credential store
+You can create a Docker credential store using `github.com/oras-project/oras-credentials-go`.
```
Comment (FeynmanZhou) [inline docs/Client_Libraries/go.mdx:247]:
```suggestion
You can create a Docker credential store using [oras-credential-go](github.com/oras-project/oras-credentials-go). 

```

Comment (FeynmanZhou) [reply docs/Client_Libraries/go.mdx:247]: @shizhMSFT Could you use a hyperlink for oras-credentials-go?

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**